ddmrp_history/security/security.xml

```diff
@@ -6,8 +6,6 @@
         <field name="name">DDMRP History Multi-Company</field>
         <field name="model_id" ref="model_ddmrp_history" />
         <field eval="True" name="global" />
-        <field
-            name="domain_force"
-        >['|',('company_id', 'in', company_ids),('company_id','=',False)]</field>
+        <field name="domain_force">[('company_id', 'in', company_ids)]</field>
```

Comment:
It makes and LGTM. Just out of curiosity. Would `[('company_id', 'in', company_ids + [False])]` be more performant than `['|',('company_id', 'in', company_ids),('company_id','=',False)]`? Could you check that?

Reply:
@LoisRForgeFlow It was actually the first thing I tried (since I did also see this syntax used in Odoo) but the performance was mostly the same or even slightly worse than with the OR operator in domain.

Unfortunately I didn't save the execution plan after settling on the actual solution after realizing company_id was mandatory anyway.

Reply:
Ok, thanks for the insights!